### PR TITLE
Update docs for initial admin and admin settings

### PR DESCRIPTION
**Pull request by WeirdMachine**

### CODE REVIEW

File: content/kubermatic/master/tutorials_howtos/administration/kkp_user/_index.en.md [hunk 1]
```diff
@@ -24,12 +24,14 @@ spec:
 
 # Initial Admin
 
-After the installation of Kubermatic Kubernetes Platform you need to set an initial admin.
-To do so make sure the account logged in once at the Kubermatic Dashboard.
+After the installation of Kubermatic Kubernetes Platform the first account that authenticates at the Dashboard is elected as an admin.
 
-Now you can edit the user with `kubectl edit user` command in the master cluster.
+The account is than capable to set admin permissions via the [dashboard]({{< ref "../admin_panel/administrators" >}}) .
```
[the remark this answers is not in the document]
Comment (embik) [reply content/kubermatic/master/tutorials_howtos/administration/kkp_user/_index.en.md:29]: @WeirdMachine it doesn't look like this section was changed but the conversation was resolved, do you disagree with the suggestion?

[the remark this answers is not in the document]
Comment (WeirdMachine) [reply content/kubermatic/master/tutorials_howtos/administration/kkp_user/_index.en.md]: I thought i applied you changes previously. Should be there now

**Review by embik — APPROVED** [review]
/lgtm

**Review by embik — APPROVED** [review]
/approve
/lgtm

**Pull request merged**